.github/workflows/DocCleanup.yml

```diff
@@ -0,0 +1,33 @@
+name: Doc Preview Cleanup
+
+on:
+  pull_request:
+    types: [closed]
+
+# Ensure that only one "Doc Preview Cleanup" workflow is force pushing at a time
+concurrency:
+  group: doc-preview-cleanup
+  cancel-in-progress: false
```

Comment:
Do we want `cancel-in-progress: true` so that we don't have multiple of this GHA running at the same time? We have this set to true for our other GHAs